externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -179,7 +179,15 @@ object SparkSQLEngine extends Logging {
   }
 
   def createSpark(): SparkSession = {
+    val engineCredentials = _sparkConf.getOption(KyuubiReservedKeys.KYUUBI_ENGINE_CREDENTIALS_KEY)
```

Comment:
this is wrong, you should get the creds from kyuubiConf

Reply:
thanks, fixed